lib/hmmlearn/base.py

```diff
@@ -143,7 +157,7 @@ def __init__(self, n_components=1,
                  n_iter=10, tol=1e-2, verbose=False,
                  params=string.ascii_letters,
                  init_params=string.ascii_letters,
-                 implementation="log"):
+                 implementation="log", max_workers=None):
```

Comment:
Needs documentation; also, the default should stay "no multiprocessing, no use of threads at all".

Reply:
I agree with @anntzer that there must be a way for users of the libary to disable threads and multiprocessing, and this should be the the default for backwards company.

There are various scenarios where someone will be using hmmlearn in a library and have some other solution to achieve parallel or concurrent computation that `hmmlearn` is not and should not be aware of. In such cases, if all cores and threads of a machine are already fully utilised, hmmlearn trying to do its own multithreading will reduce performance.